[Issue 10990] Fix the default value of httpServerPort in reference conf (#10992)
apache/pulsar: site2/docs/reference-configuration.md

|zkRetryBackoffMaxMs|The maximum time that the Zookeeper client backoff retries in milliseconds.|10000|
|zkEnableSecurity|Set ACLs on every node written on ZooKeeper, allowing users to read and write BookKeeper metadata stored on ZooKeeper. In order to make ACLs work you need to setup ZooKeeper JAAS authentication. All the bookies and Client need to share the same user, and this is usually done using Kerberos authentication. See ZooKeeper documentation.|false|
|httpServerEnabled|The flag enables/disables starting the admin http server.|false|

> |httpServerPort|The http server port to listen on. By default, the value is 8080. Use `8000` as the port to keep it consistent with prometheus stats provider.|8080

Reviewer: ```suggestion
|httpServerPort|The HTTP server port to listen on. By default, the value is `8080`. If you want to keep it consistent with the Prometheus stats provider, you can set it to `8000`.|8080
```

Do you mean this?

Author: Yes, I have addressed your comment, please take a look again.
